README.md

```diff
@@ -384,22 +322,50 @@ Use the `configlet` tool to generate a README from shared metadata:
 
 If adding a new exercise:
 
-* a [generator](#implementing-a-generator) should be implemented.
-* a minimal, partial, solution should be able to be pushed, in order to create a
-  WIP pull request.
+* a [generator][#implementing-a-generator] should be implemented.
+* a minimal, partial, solution should be able to be pushed, in order to create a WIP pull request.
 
-For an in-depth discussion of how exercism language tracks and exercises work,
-please see the
-[contributing guide](https://github.com/exercism/x-api/blob/master/CONTRIBUTING.md#the-exercise-data).
+For an in-depth discussion of how exercism language tracks and exercises work, please see the [contributing guide][contributing guide].
 
-If you're just getting started and looking for a helpful way to get involved,
-take a look at
-[regenerating the test suites](#regenerating-a-test-suite),
-[porting an exercise from another language](https://github.com/exercism/docs/blob/master/you-can-help/implement-an-exercise-from-specification.md),
-or [creating an automated test generator](#implementing-a-generator).
+If you're just getting started and looking for a helpful way to get involved, take a look at [regenerating the test suites][#regenerating-a-test-suite], [porting an exercise from another language][port exercise], or [creating an automated test generator][#implementing-a-generator].
 
 ## Ruby icon
+
 The Ruby icon is the Vienna.rb logo, and is used with permission. Thanks Floor
 Dress :)
 
+[#anatomy-of-an-exercise]: #anatomy-of-an-exercise
+[assertion]: https://github.com/exercism/ruby/blob/master/lib/generator/exercise_case/assertion.rb
+[#canonical-data]: #canonical-data
+[canonical data]: https://github.com/exercism/problem-specifications/tree/master/exercises
+[canonical exercises]: https://github.com/exercism/problem-specifications/tree/master/exercises
+[#changing-a-generated-exercise]: #changing-a-generated-exercise
+[configlet-workflow-badge]: https://github.com/exercism/ruby/workflows/Configlet%20CI/badge.svg
+[configlet-workflow]: https://github.com/exercism/ruby/actions?query=workflow%3A%22Configlet+CI%22+branch%3Amaster
```

Comment:
I've changed the links to use the badge links that GitHub suggests (the second like was also using the deprecated master branch)

```suggestion
[configlet-workflow-badge]: https://github.com/exercism/ruby/actions/workflows/configlet.yml/badge.svg
[configlet-workflow]: https://github.com/exercism/ruby/actions/workflows/configlet.yml
```